deploy/crds/crd-certificates.yaml

```diff
@@ -229,6 +229,18 @@ spec:
                       type: array
                       items:
                         type: string
+                    extraNames:
+                      description: ExtraNames is a list of pkix.ExtraNames to be set on the Certificate.
```

Comment:
From a user's perspective, what is a pkix.ExtraNames? Is there a more general (ie not go specific) term to use here?

Reply:
We could name it something like SubjectExtraFields? I named it this way so it was clear how it maps to the underlying golang pkix library. Otherwise, someone trying to understand the code would need to know it changes from SubjectExtraNames to pkix.ExtraNames. I'm not opposed to it, but I'm not sure if that makes it clearer or not.